File: Computer-Seekho-Java/ComputerSeekho/src/main/java/com/Project/DTO/PaymentDTO.java
package com.Project.DTO;

import java.time.LocalDate;

import lombok.Data;

@Data
public class PaymentDTO {
    private String studentName;
    private String studentEmail;
    private int amount;
    private LocalDate paymentDate;
    private String paymentTypeDesc;

    public PaymentDTO(String studentName, String studentEmail, int amount, LocalDate paymentDate, String paymentTypeDesc) {
        this.studentName = studentName;
        this.studentEmail = studentEmail;
        this.amount = amount;
        this.paymentDate = paymentDate;
        this.paymentTypeDesc = paymentTypeDesc;
    }

    // // Getters & Setters
    // public String getStudentName() { return studentName; }
    // public String getStudentEmail() { return studentEmail; }
    // public int getAmount() { return amount; }
    // public LocalDate getPaymentDate() { return paymentDate; }
    // public String getPaymentTypeDesc() { return paymentTypeDesc; }
}

File: Computer-Seekho-.NET/Computer_Seekho_DN/DTO/PaymentDTO.cs
namespace Computer_Seekho_DN.DTO;

public class PaymentDTO
{
    public string StudentName { get; set; }
    public string StudentEmail { get; set; }
    public int Amount { get; set; }
    public DateOnly PaymentDate { get; set; }
    public string PaymentTypeDesc { get; set; }

    public PaymentDTO(string studentName, string studentEmail, int amount, DateOnly paymentDate, string paymentTypeDesc)
    {
        StudentName = studentName;
        StudentEmail = studentEmail;
        Amount = amount;
        PaymentDate = paymentDate;
        PaymentTypeDesc = paymentTypeDesc;
    }
}
